documentation/concepts.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Concepts",
+  "order": 10,
```

Comment:
[minor] I would set order to `6` to place this page after the "Best practices and patterns" page, and update the order value of "Linting" and "Releasing" accordingly.